<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index</class>
 <widget class="QWidget" name="Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1057</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="video_region">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1520</width>
     <height>1037</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="right_control" native="true">
   <property name="geometry">
    <rect>
     <x>1580</x>
     <y>10</y>
     <width>320</width>
     <height>1037</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_left_m">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_right_m">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>530</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_down_m">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_up_m">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_reset_m">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>恢复</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_data_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据显示</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="open_control">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连接设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startReceiveData_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始接收数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="light_control_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>灯光</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>22</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="horizontal_m" name="mq2_h" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>200</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="horizontal_m" name="mq135" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>200</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="horizontal_m" name="dht11_w" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>200</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mq2_mpp_la">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>可燃气：</string>
    </property>
   </widget>
   <widget class="QLabel" name="mq135_la">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>空气质量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="dht11_shi_la">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="video_show_la">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>视频栏</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="con_lan_la">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>控制栏</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bottom_la">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="data_bar_label" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1520</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QLabel" name="Time_my">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MQ_135">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>空气质量：？？？</string>
    </property>
   </widget>
   <widget class="QLabel" name="DHT11_btn_H">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度：？？？</string>
    </property>
   </widget>
   <widget class="QLabel" name="MQ2_ppm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>可燃气浓度：？？？</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>horizontal_m</class>
   <extends>QWidget</extends>
   <header location="global">horizontal_m.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
